README.md

```diff
@@ -16,6 +16,23 @@ for more background, objectives, key use cases.
 
 ## Development
 
+### Docker
+
+You may run the Breach Alerts service with [Docker](https://www.docker.com/).
+
+If you have Docker installed, you can:
+
+1. Build the container image:
+
+    `docker build -t blurts-server .`
+    
+2. Run the container:
+
+    `docker run -d -p 6060:6060 blurts-server`
+
+Otherwise, the instructions below explain how to Install and run Breach Alerts
+directly on your machine.
+
 ### Requirements
 
 * [Node](https://nodejs.org/) 9 (with npm)
```

Comment:
Let's go with LTS in `package.json` and `Dockerfile` since FxA is using 8.x (i.e., LTS) branch.

We can let Travis CI keep building on lts and current so we can get an alert if we're doing something that breaks in current.